src/SignalR/server/SignalR/test/HubConnectionHandlerTestUtils/Hubs.cs

```diff
@@ -23,7 +23,7 @@ public Task ClientSendMethod(string userId, string message)
             return Clients.User(userId).SendAsync("Send", message);
         }
 
-        public Task SendToMultipleUsers(IReadOnlyList<string> userIds, string message)
+        public Task SendToMultipleUsers(List<string> userIds, string message)
```

Comment:
Is this change unrelated to the port?

Reply:
Nope, `IReadOnlyList<T>` isn't supported yet

Reply:
Is there a skipped test for this?

Reply:
No